src/pages/resultsView/enrichments/GenericAssayEnrichmentsDataSetDropdown.tsx

```diff
@@ -0,0 +1,105 @@
+import * as React from 'react';
```

Comment:
This too, looks very much similar to `EnrichmentsDataSetDropdown.tsx`. combine them?